addon/adapters/user.js

```diff
@@ -1,4 +1,20 @@
 import OsfAdapter from './osf-adapter';
+import Ember from 'ember';
 
 export default OsfAdapter.extend({
+    findHasMany(store, snapshot, url, relationship) {
```

Comment:
So to clarify, I assume the key problem is that we use two embeds?
`?embed=parent&embed=root'`

... and thus it doesn't work with a query object of form `{embed: 'parent', embed:'root'}` ?

Realistically, this seems like an API limitation that we'll have to keep hacking around in the future- this won't be the last time we need more than one embed in a request. Sometimes when a single parameter has multiple values, I've see APIs accept a comma-separated list of values: `embed=parent,root`. That seems in keeping with the JSONAPI spec, eg 
http://jsonapi.org/format/#fetching-includes

@brianjgeiger , do you have any thoughts on the best way to support multiple values for a single query param in the API going forward?

 I've [confirmed](https://staging-api.osf.io/v2/users/me/nodes/?embed=parent,root) that our API doesn't support that for the embed parameter at present.